### DISCUSSION TITLE
I have a problem with PowerSystems library.

### DISCUSSION THREAD

**Discussion by Stur-frog** [OpenModelica/OpenModelica#11338] [Q&A]
I want to change the model presented in PowerSystems.PowerWorld.PowerWorld, the changes consist in removing the HydroPlant and adding an additional energy generator, for example, a solar plant, but when the HydroPlant is removed and the simulation is repeated, the system indicates the following message: Simulation call terminate() at time 4.206814 Message: FREQUENCY EXCEEDS BOUNDS! And I do not understand why this message is issued, although the production and consumption values ​​indicated in the table, I have selected so that it satisfies the system requirements. If you have any ideas or have had experience with this model, I would appreciate your help.
![image](https://github.com/OpenModelica/OpenModelica/assets/131712807/0f647d8f-4b02-46dd-ba9d-7e4a69150b53)

**Comment by casella** [comment]
This is an islanded system, so if the injected active power generation does not match the active power consumption + losses, the result is an acceleration/deceleration of the rotating generators, until eventually the frequency bounds are exceeded. At that point in real life machines will start disconnecting from the grid for safety, and a blackout would ensue shortly.

**Comment by Stur-frog** [comment]
I agree with you, I need to have enough power to cover consumption and losses. But if I selected such production values that compensate for the lack of a HydroPlant, the system still does not work and gives incorrect data. For example, the power of the generator of the powerPlant shows a value of 11,000 MW instead of 700 MW.
What can you advise me to solve this problem?
![image](https://github.com/OpenModelica/OpenModelica/assets/131712807/37c84af1-e0f9-4e2e-bbb6-da853785c88b)

**Comment by casella** [comment]
In that case I would suggest to investigate how those 11000 MW are distributed throughout the system. I guess you are getting much higher voltages than needed. How is voltage handled in your models? Do you have active voltage control on the grid?
